Aidyn ReChronicled: copyTextureRegion from a texture into a buffer on Vulkan (vkCmdCopyImageToBuffer), for reading
back save pictures (tools/patches/rt64-save-pictures.patch, src/game/save_pictures.cpp).

diff --git i/plume_vulkan.cpp w/plume_vulkan.cpp
index f1dd503..83120b3 100644
--- i/plume_vulkan.cpp
+++ w/plume_vulkan.cpp
@@ -3229,6 +3229,29 @@ namespace plume {
             imageCopy.imageExtent.depth = srcLocation.placedFootprint.depth;
             vkCmdCopyBufferToImage(vk, srcBuffer->vk, dstTexture->vk, toImageLayout(dstTexture->textureLayout), 1, &imageCopy);
         }
+        // Aidyn ReChronicled (tools/patches/rt64-save-pictures.patch): a texture's area into a buffer, for reading back.
+        else if ((dstLocation.type == RenderTextureCopyType::PLACED_FOOTPRINT) && (srcLocation.type == RenderTextureCopyType::SUBRESOURCE)) {
+            assert(dstBuffer != nullptr);
+            assert(srcTexture != nullptr);
+
+            VkBufferImageCopy imageCopy = {};
+            imageCopy.bufferOffset = dstLocation.placedFootprint.offset;
+            imageCopy.bufferRowLength = dstLocation.placedFootprint.rowWidth;
+            imageCopy.bufferImageHeight = dstLocation.placedFootprint.height;
+            imageCopy.imageSubresource.aspectMask = toAspectFlags(srcTexture->desc.format, srcTexture->desc.flags);
+            imageCopy.imageSubresource.baseArrayLayer = srcLocation.subresource.arrayIndex;
+            imageCopy.imageSubresource.layerCount = 1;
+            imageCopy.imageSubresource.mipLevel = srcLocation.subresource.mipLevel;
+            if (srcBox != nullptr) {
+                imageCopy.imageOffset = { srcBox->left, srcBox->top, srcBox->front };
+                imageCopy.imageExtent = { uint32_t(srcBox->right - srcBox->left), uint32_t(srcBox->bottom - srcBox->top), uint32_t(srcBox->back - srcBox->front) };
+            }
+            else {
+                imageCopy.imageExtent = { dstLocation.placedFootprint.width, dstLocation.placedFootprint.height, dstLocation.placedFootprint.depth };
+            }
+
+            vkCmdCopyImageToBuffer(vk, srcTexture->vk, toImageLayout(srcTexture->textureLayout), dstBuffer->vk, 1, &imageCopy);
+        }
         else {
             VkImageCopy imageCopy = {};
             imageCopy.srcSubresource.aspectMask = toAspectFlags(srcTexture->desc.format, srcTexture->desc.flags);
